db/migrations/1721779124878-migration.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Migration1721779124878 implements MigrationInterface {
+  name = 'Migration1721779124878';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "abstract_base_entity" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "created_at" TIMESTAMP NOT NULL DEFAULT now(), "updated_at" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "PK_723c213727b1038d476972ce2f2" PRIMARY KEY ("id"))`
```

Comment:
Remove this migration, abstract base entity is just an abstract entity and does not need a table in the database, It was created to be extended by other entities.

Reply:
Removed query